include/NovelRT/ResourceManagement/Desktop/DesktopResourceLoader.h

```diff
@@ -15,8 +15,12 @@ namespace NovelRT::ResourceManagement::Desktop
     private:
         LoggingService _logger;
 
+    protected:
+        void WriteAssetDatabaseFile() final;
+        void LoadAssetDatabaseFile() override;
+
     public:
-        [[nodiscard]] TextureMetadata LoadTextureFromFile(std::filesystem::path filePath) final;
+        [[nodiscard]] TextureMetadata LoadTexture(std::filesystem::path filePath) final;
```

Comment:
Is this to be an example for future file loading for resources? More of a curiosity question

Reply:
This was more to bring it in line with the other resource loading methods, it looked weird. :D

Reply:
No worries - Audio is currently considered a Resource but the loading method is not here, so this was more of a "do we have a future TODO" q